packages/mobile/src/paymentRequest/PaymentRequestConfirmation.tsx

```diff
@@ -100,7 +95,8 @@ class PaymentRequestConfirmation extends React.Component<Props> {
       requesterE164Number: this.props.e164PhoneNumber,
       requesteeAddress,
       currency: currencyToShortMap[CURRENCY_ENUM.DOLLAR],
-      comment: reason,
+      // NOTE: Add this back in when redesigning this screen
+      comment: 'placeholder',
```

Comment:
Should this be set back to reason?

Reply:
Not yet, I am going to do resdesign for the payment screens in the next PR